ColorPicker/ColorPicker.js

```diff
@@ -0,0 +1,397 @@
+			<Row className={css.colorsRow} wrap>
+				{presetColors?.map((presetColor, presetColorIndex) => {
+
+					return (
+						<Cell key={presetColor + '-' + presetColorIndex} size="25%">
+							<SpottableButton
+								id={presetColor}
+								className={css.coloredButton}
+								minWidth={false}
+								onClick={handleClick}
+								style={{backgroundColor: presetColor}}
+								type="color"
+							/>
+						</Cell>
+					);
+				})}
+			</Row>
+			<div>
+				<Column className={css.colorPickerSliders}>
+					<BodyText className={css.colorSliderText} css={css}>Hue {hue}</BodyText>
+					<Slider
+						className={css.colorSlider}
+						max={356}
+						min={0}
+						onBlur={onSliderValueChange}
+						onClick={onSliderValueChange}
+						onChange={changeHue}
+						value={hue}
+					/>
+					<BodyText className={css.colorSliderText} css={css}>Saturation %{saturation}</BodyText>
+					<Slider
+						className={css.colorSlider}
+						max={100}
+						min={0}
+						onBlur={onSliderValueChange}
+						onClick={onSliderValueChange}
+						onChange={changeSaturation}
+						value={saturation}
+					/>
+					<BodyText className={css.colorSliderText} css={css}>Lightness %{lightness}</BodyText>
+					<Slider
+						className={css.colorSlider}
+						max={100}
+						min={0}
+						onBlur={onSliderValueChange}
+						onClick={onSliderValueChange}
+						onChange={changeLightness}
+						value={lightness}
+					/>
+				</Column>
+				<div className={css.coloredDiv} style={{backgroundColor: `hsl(${hue} ,${saturation}%, ${lightness}%)`}} />
+			</div>
+		</Cell>
+	);
+};
+
+PopupContent.propTypes = {
+	color: PropTypes.string,
+	colorHandler: PropTypes.func,
+	css: PropTypes.object,
+	presetColors: PropTypes.array
+};
+
+/**
+ * A color picker component.
+ *
+ * This component is most often not used directly but may be composed within another component as it
+ * is within [ColorPicker]{@link sandstone/ColorPicker}.
+ *
+ * @class ColorPickerBase
+ * @memberof sandstone/ColorPicker
+ * @ui
+ * @private
+ */
+
+const ColorPickerBase = kind({
+	name: 'ColorPicker',
+
+	functional: true,
+
+	propTypes: /** @lends sandstone/ColorPicker.ColorPickerBase.prototype */ {
+
+		/**
+		 * Indicates the color.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		color: PropTypes.string,
+
+		/**
+		 * Called when color is modified.
+		 *
+		 * @type {Function}
+		 * @public
+		 */
+		colorHandler: PropTypes.func,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * `colorPicker` - The root class name
+		 * `coloredDiv`  - A class name used for a single div
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		css: PropTypes.object,
+
+		/**
+		 * Applies the `disabled` class.
+		 *
+		 * When `true`, the color picker is shown as disabled.
+		 *
+		 * @type {Boolean}
+		 * @default false
```

Comment:
you mentioned a default value in the jsdoc for multiple props, but i don;t see the defaultValue block in the components. Can you double check?